### ISSUE TITLE
[Question] How to activate extra pane for default (aka 'features') workflow, e.g. dictionary

### ISSUE THREAD

**Issue by astr0gator** [mr-pennyworth/alfred-extra-pane#29]
On your screenshot you enhanced Dictionary app. Did you recreate the functionality with a custom workflow or somehow extended pane for default workflows ('features', as Alfred calls them)? A snippet would be ideal

**Comment by mr-pennyworth** [comment]
yup, it is a custom workflow: https://github.com/mr-pennyworth/alfred-better-dictionaries
at the moment, native alfred features aren't supported
